thyrel-web/src/hooks/SessionProvider.tsx

```diff
@@ -27,6 +30,31 @@ export function SessionContextProvider({
   const { websocket } = useWebsocketContext();
   const { enqueueSnackbar } = useSnackbar();
 
+  console.log(currentElement);
+  function onSave(content: string) {
+    const elementId = currentElement?.id;
+    const url = `element/${elementId}`;
+
+    const elementContent =
+      currentElement?.type == 1 ? { drawimage: content } : { text: content };
```

Comment:
```suggestion
      currentElement?.type === 1 ? { drawimage: content } : { text: content };
```
Attention
petit exemple de pourquoi 👍 
```js
1 == 1 // true
1 === 1 // true
1 === "1" // false
1 == "1" // true
```